cli/bin/grain.js

```diff
@@ -1,4 +1,5 @@
-#!/usr/bin/env node
+#!/bin/sh
+// 2>/dev/null; exec /usr/bin/env node --experimental-wasm-bigint "$0" "$@"
```

Comment:
?

Reply:
Heh. Basically, we need the `experimental-wasm-bigint` flag for @wasmer/wasi to properly pass int64s to/from JS, but Linux machines don't allow you to pass more than one argument to a shebang.

https://unix.stackexchange.com/questions/65235/universal-node-js-shebang/65295#65295

This is temporary until it's not experimental anymore.

Reply:
Gotcha, that makes sense. Maybe a comment to explain so that we don't forget?